### PR TITLE
[ICA] Smoke Alarm - 3407

### PR DESCRIPTION
**Pull request by QingAn**
Smoke Alarm object

### CODE REVIEW

File: 3407.xml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+The above license is used as a license under copyright only. Please
+reference the company patent licensing terms:
+www.ica-alliance.org 
+-->
+<LWM2M xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://www.openmobilealliance.org/tech/profiles/LWM2M-v1_1.xsd">
+	<Object ObjectType="MODefinition">
+		<Name>Smoke Alarm</Name>
+		<Description1><![CDATA[This is an alarm that should be raised if the meter detects CO2 concentration above a pre-configured threshold.]]></Description1>
+		<ObjectID>3407</ObjectID>
+		<ObjectURN></ObjectURN>
```
Comment (jpradocueva) [inline 3407.xml]: @QingAn URN value should be: urn:oma:lwm2m:ext:3407. We are assuming that you are submitting this object on behalf of ICA Alliance.

**Review by akeranen — CHANGES_REQUESTED** [review]
See comments in #202

**Comment by jpradocueva** [comment]
This PR has failed TS/DDF validation:


Error | Time | Description | ObjectID and/or ResourceID
-- | -- | -- | --
401 |   | Object URN missmatch | 3407
463 |   | RangeEnumeration is different | 3407:6044
464 |   | Units is different | 3407:6044
463 |   | RangeEnumeration is different | 3407:6046
460 |   | Name is different | 3407:6047
463 |   | RangeEnumeration is different | 3407:6047
460 |   | Name is different | 3407:6048

File: 3407.xml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+		<MultipleInstances>Multiple</MultipleInstances>
+		<Mandatory>Optional</Mandatory>
+		<Resources>
+			<Item ID="6044">
+				<Name>Battery Percentage</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration>0.00-100.00</RangeEnumeration>
```
Comment (seanmcilroy29) [inline 3407.xml]: Update RangeEnumeration to 0.00..100.00

- Leave blank

File: 3407.xml [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+				<Units>%</Units>
```
Comment (seanmcilroy29) [inline 3407.xml]: Remove %

- Leave blank

File: 3407.xml [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+				<Units>dBW</Units>
+				<Description><![CDATA[Received Signal Strength Indicator]]></Description>
+			</Item>
+			<Item ID="6046">
+				<Name>Smoke Sensor State</Name>
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Mandatory</Mandatory>
+				<Type>Boolean</Type>
+				<RangeEnumeration>Normal (0) or Smoke detected (1)</RangeEnumeration>
```
Comment (seanmcilroy29) [inline 3407.xml]: Remove ``Normal (0) or Smoke detected (1)`` from Range Enumeration
- Leave empty

File: 3407.xml [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+				<Units></Units>
+				<Description><![CDATA[Indicate current smoke sensor state]]></Description>
+			</Item>
+			<Item ID="6047">
+				<Name>CO2</Name>
```
Comment (seanmcilroy29) [inline 3407.xml]: Change name to ``Detected CO2 percentage``

File: 3407.xml [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+				<Operations>R</Operations>
+				<MultipleInstances>Single</MultipleInstances>
+				<Mandatory>Optional</Mandatory>
+				<Type>Float</Type>
+				<RangeEnumeration>0.00-100.00</RangeEnumeration>
```
Comment (seanmcilroy29) [inline 3407.xml]: Change Range Enumeration to ``0.00..100.00``

File: 3407.xml [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+				<Units>%</Units>
+				<Description><![CDATA[Current detected CO2 percentage]]></Description>
```
Comment (seanmcilroy29) [inline 3407.xml]: Change description to ``Current level of detected CO2 as a percentage (of what?)``  

``(of what?)`` - Please Specify the text that should be applied here.

File: 3407.xml [hunk 8]
```diff
@@ -0,0 +1,97 @@
...
+			</Item>
+			<Item ID="6048">
+				<Name>Decibel Value</Name>
```
Comment (seanmcilroy29) [inline 3407.xml]: Change Name to ``Alarm loudness``

File: 3407.xml (hunk 1 above)
Comment (seanmcilroy29) [inline 3407.xml]: Add objectURN ``urn:oma:lwm2m:ext:3407``

**Comment by seanmcilroy29** [comment]
@QingAn - Reusable Resources (RR) 6044, 6045, 6046, 6047, 6048 have been agreed by the WG.

To reflect the Agreed RRs, Comments have been applied against your submitted Object (3407) that requires you to apply directly.

**Review by jpradocueva — CHANGES_REQUESTED** [review]

**Comment by mkgillmore** [comment]
@QingAn Please note that the IPSO working group has decided that we will close this pull request within 30 days due to inactivity.   Your object will not officially be registered unless you help resolve the problems found with the original PR.  11/14/2019

File: 3407.xml (hunk 7 above)
Comment (judy-zhuhr) [reply 3407.xml]: Current level of detected CO2 as a percentage of concentration.

File: 3407.xml (hunk 3 above)
Comment (QingAn) [reply 3407.xml]: agree

File: 3407.xml (hunk 4 above)
Comment (QingAn) [reply 3407.xml]: agree

File: 3407.xml (hunk 5 above)
Comment (QingAn) [reply 3407.xml]: agree

File: 3407.xml (hunk 6 above)
Comment (QingAn) [reply 3407.xml]: agree

File: 3407.xml (hunk 7 above)
Comment (QingAn) [reply 3407.xml]: Current level of detected CO2 as a percentage of concentration

File: 3407.xml (hunk 8 above)
Comment (QingAn) [reply 3407.xml]: agree

File: 3407.xml (hunk 1 above)
Comment (QingAn) [reply 3407.xml]: agree

Comment (QingAn) [reply 3407.xml]: Yes, it is on behalf of ICA.

**Comment by QingAn** [comment]
I have applied all the comments.

**Comment by mkgillmore** [comment]
Group agrees to merge 12/19/19 given the full stops are added to the descriptions

**Pull request merged**